guides/kubernetes.md

```diff
@@ -0,0 +1,136 @@
+# Kubernetes guidance
+
+## Configure NGINX Ingress Controller
+An [Ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) is an application that runs in a Kubernetes cluster and configures an HTTP load balancer according to Ingress resources.
+
+In the case of NGINX, the Ingress controller is deployed in a pod along with
+the load balancer.
+
+### Installation
+The documentation for [NGINX's chart](https://github.com/helm/charts/blob/master/stable/nginx-ingress/README.md) includes details on how to install it.
```

Comment:
The linked resource has been deprecated, and perhaps should link to https://github.com/kubernetes/ingress-nginx instead

Reply:
Agreed.  I've updated the link and the following TL;DR install guide.

Coincidentally I discovered that this weekend when I was making changes to NGINX in my own cluster, you now get hammered with deprecation warnings.